src/forms/NumberInput.jsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import cx from 'classnames';
+import Flex from '../layout/Flex';
+import FlexItem from '../layout/FlexItem';
+import Icon from '../media/Icon';
+
+export const DECREMENT_BTN_CLASS = 'decrementButton';
+export const INCREMENT_BTN_CLASS = 'incrementButton';
+
+/**
+ * @module NumberInput
+ */
+class NumberInput extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			value: props.value || '',
+		};
+
+		this.decrement = this.decrement.bind(this);
+		this.increment = this.increment.bind(this);
+		this.onBlur = this.onBlur.bind(this);
+		this.onChange = this.onChange.bind(this);
+		this.onFocus = this.onFocus.bind(this);
+	}
+
+	onBlur(e) {
+		if (document.activeElement !== this.fauxInputEl && document.activeElement !== this.decrementBtnEl && document.activeElement !== this.incrementBtnEl) {
```

Comment:
minor nitpck... you could either split this expression into multiple lines at each `&&`, or you could do something like this:

```
const formControls = [
	this.fauxInputEl,
	this.decrementBtnEl,
	this.incrementBtnEl
];

if (formControls.every(c => c !== document.activeElement)) {
	...
}
```

cc @mmcgahan because I'm curious to hear his opinion